tools/matchms/matchms_metadata_match.xml

```diff
@@ -0,0 +1,91 @@
+<tool id="matchms_metadata_match" name="matchms metadata match" version="@TOOL_VERSION@+galaxy0" python_template_version="3.8">
+    <description>matchms metadata match calculation for numeric fields based on tolerance</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator"/>
+
+    <requirements>
+        <requirement type="package" version="1.1.4">pandas</requirement>
+        <requirement type="package" version="0.55.1">numba</requirement>
+        <requirement type="package" version="@TOOL_VERSION@">matchms</requirement>
+    </requirements>
+
+    <environment_variables>
+        <environment_variable name="MPLCONFIGDIR">/tmp</environment_variable>
```

Comment:
please do not do that, use $TEMP or `\$_GALAXY_JOB_TMP_DIR` instead, so an admin can manipulate that setting from outside